recipes/z3/all/conanfile.py

```diff
@@ -8,7 +9,7 @@
 class Z3Conan(ConanFile):
     name = "z3"
     description = "The Z3 Theorem Prover"
-    topics = ("z3", "theorem", "SMT", "satisfiability", "prover", "solver")
+    topics = ("z3", "theorem", "satisfiability modulo theories", "satisfiability", "prover", "solver")
```

Comment:
```suggestion
    topics = ("z3", "theorem", "smt", "satisfiability", "prover", "solver")
```
Keep the previous, it's more popular and easier to find on Github topics.